packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -290,9 +290,25 @@ export function setDynamicConfigValue(config: FieldConfig, value: DynamicConfigV
   }
 }
 
+/**
+ * Merge the links from the data source config and the panel config
+ * @param dataSourceLinks coming from the datasource
+ * @param defaultLinks coming from the panel config
+ * @returns merged links
+ */
+
+function mergeLinks(dataSourceLinks: DataLink[], defaultLinks: DataLink[]) {
+  // Combine the config links and the default links
+  return [...dataSourceLinks, ...defaultLinks];
+}
+
```

Comment:
This function does not make much sense to be defined. Let's just add the spreads inline, shall we? With an appropriate comment.